2024-01-26 15:31:00 ERROR Elements not present after 0.5 seconds on page: https://capital.com/es/what-is-cfd-trading
2024-01-26 15:31:00 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/SeleniumCapital/SeleniumCapital/pages/base_page.py", line 82, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/SeleniumCapital/SeleniumCapital/pages/base_page.py", line 293, in element_is_visible
    return Wait(self.browser, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-cIj6ROMf-py3.11/lib/python3.11/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x55a8dcbf8f83 <unknown>
#1 0x55a8dc8b1cf7 <unknown>
#2 0x55a8dc90199e <unknown>
#3 0x55a8dc901aa1 <unknown>
#4 0x55a8dc94cd64 <unknown>
#5 0x55a8dc92b0dd <unknown>
#6 0x55a8dc94a006 <unknown>
#7 0x55a8dc92ae53 <unknown>
#8 0x55a8dc8f2dd4 <unknown>
#9 0x55a8dc8f41de <unknown>
#10 0x55a8dcbbd531 <unknown>
#11 0x55a8dcbc1455 <unknown>
#12 0x55a8dcba9f55 <unknown>
#13 0x55a8dcbc20ef <unknown>
#14 0x55a8dcb8d99f <unknown>
#15 0x55a8dcbe6008 <unknown>
#16 0x55a8dcbe61d7 <unknown>
#17 0x55a8dcbf8124 <unknown>
#18 0x7f1b4f094ac3 <unknown>

2024-01-26 15:31:06 ERROR Elements not present after 0.5 seconds on page: https://capital.com/es/what-is-cfd-trading
2024-01-26 15:31:06 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/SeleniumCapital/SeleniumCapital/pages/base_page.py", line 82, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/SeleniumCapital/SeleniumCapital/pages/base_page.py", line 293, in element_is_visible
    return Wait(self.browser, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-cIj6ROMf-py3.11/lib/python3.11/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x55a8dcbf8f83 <unknown>
#1 0x55a8dc8b1cf7 <unknown>
#2 0x55a8dc90199e <unknown>
#3 0x55a8dc901aa1 <unknown>
#4 0x55a8dc94cd64 <unknown>
#5 0x55a8dc92b0dd <unknown>
#6 0x55a8dc94a006 <unknown>
#7 0x55a8dc92ae53 <unknown>
#8 0x55a8dc8f2dd4 <unknown>
#9 0x55a8dc8f41de <unknown>
#10 0x55a8dcbbd531 <unknown>
#11 0x55a8dcbc1455 <unknown>
#12 0x55a8dcba9f55 <unknown>
#13 0x55a8dcbc20ef <unknown>
#14 0x55a8dcb8d99f <unknown>
#15 0x55a8dcbe6008 <unknown>
#16 0x55a8dcbe61d7 <unknown>
#17 0x55a8dcbf8124 <unknown>
#18 0x7f1b4f094ac3 <unknown>